doc/releasenotes/15_0_0_summary.md

```diff
@@ -79,10 +80,10 @@ connections, similar to the way pooling happens for write buffers. Defaults to o
 
 We introduced the ability to resume a VDiff2 workflow:
 ```
-$ vtctlclient --server=localhost:15999 VDiff -- --v2 customer.commerce2customer resume 4c664dc2-eba9-11ec-9ef7-920702940ee0
+$ vtctlclient --server=localhost:15999 VDiff --v2 customer.commerce2customer resume 4c664dc2-eba9-11ec-9ef7-920702940ee0
```

Comment:
Why remove the double dashes? Compare the output of these two things for an example of why I think we should always use them and why I feel the way I do about noting that it's (virtually) required now in the release notes:
```
$ vtctlclient --server=localhost:15999 VDiff --help
$ vtctlclient --server=localhost:15999 VDiff -- --help
```
This same comment applies to all of the other places we remove them in the PR.

Reply:
because that gets in the way of moving to cobra (cobra does one full-blown parse by merging together the parent/subcommand flag-sets, so if you have the `--` after the sub-command name then all the sub-command flags get treated as positional arguments instead of flags)